test/unit/org/apache/cassandra/cql3/statements/TransactionStatementTest.java

```diff
@@ -33,14 +33,7 @@
 import org.apache.cassandra.transport.messages.ResultMessage;
 import org.assertj.core.api.Assertions;
 
-import static org.apache.cassandra.cql3.statements.TransactionStatement.DUPLICATE_TUPLE_NAME_MESSAGE;
-import static org.apache.cassandra.cql3.statements.TransactionStatement.EMPTY_TRANSACTION_MESSAGE;
-import static org.apache.cassandra.cql3.statements.TransactionStatement.ILLEGAL_RANGE_QUERY_MESSAGE;
-import static org.apache.cassandra.cql3.statements.TransactionStatement.INCOMPLETE_PRIMARY_KEY_SELECT_MESSAGE;
-import static org.apache.cassandra.cql3.statements.TransactionStatement.NO_CONDITIONS_IN_UPDATES_MESSAGE;
-import static org.apache.cassandra.cql3.statements.TransactionStatement.NO_COUNTERS_IN_TXNS_MESSAGE;
-import static org.apache.cassandra.cql3.statements.TransactionStatement.NO_TIMESTAMPS_IN_UPDATES_MESSAGE;
-import static org.apache.cassandra.cql3.statements.TransactionStatement.SELECT_REFS_NEED_COLUMN_MESSAGE;
+import static org.apache.cassandra.cql3.statements.TransactionStatement.*;
```

Comment:
please revert, this goes against our style guide; we don't like `.*`

Reply:
Done, but our `checkstyle.xml` does not include such a rule. Where can I find the evidence for your claim?

Reply:
https://cassandra.apache.org/_/development/code_style.html is what defines our style, but it overrides https://www.oracle.com/java/technologies/javase/codeconventions-fileorganization.htm

In both cases I don't see this documented... so I think we only approved moving away from `.*` in a mailing list thread and didn't update our guides?  Sorry about that =(

The only place that hits at that is `ide/idea/codeStyleSettings.xml` which makes it 100 elements before we replace with `.*`